spec/requests/discount_request_spec.rb

```diff
@@ -19,13 +19,26 @@
       fetch_discount_requests
 
       expect(response).to have_http_status(:ok)
-      expect(json).to eq(serialize_all(discount_requests))
+      expect(json).to eq(serialize_all(discount_requests.sort_by(&:created_at).reverse))
     end
 
     it 'does not return other company discount requests' do
       fetch_discount_requests
 
       expect(json).not_to include(serialize(other_discount_requests.first))
     end
+
+    context 'when company not found' do
+      subject(:fetch_discount_requests) do
+        get company_discount_requests_path(0), headers: headers
```

Comment:
```suggestion
        get company_discount_requests_path(0), headers: headers
```

How about `Faker::Internet.uuid` instead of 0 here

Reply:
Nice, I've used `FFaker::Number.number` since `FFaker::Internet.uuid` is not valid :/